sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/util/CombineFnUtilTest.java

```diff
@@ -59,4 +62,15 @@ public void testNonSerializable() throws Exception {
     ObjectOutputStream oos = new ObjectOutputStream(buffer);
     oos.writeObject(CombineFnUtil.bindContext(mockCombineFn, StateContexts.nullContext()));
   }
+
+  @Test
+  public void testToFnWithContext() throws Exception {
+    CombineFnWithContext<Integer, int[], Integer> fnWithContext =
+        CombineFnUtil.toFnWithContext(new Sum.SumIntegerFn());
+    assertTrue(fnWithContext == CombineFnUtil.toFnWithContext(fnWithContext));
+
+    KeyedCombineFnWithContext<Object, Integer, int[], Integer> keyedFnWithContext =
+        CombineFnUtil.toFnWithContext(new Sum.SumIntegerFn().asKeyedFn());
+    assertTrue(keyedFnWithContext == CombineFnUtil.toFnWithContext(keyedFnWithContext));
```

Comment:
This is only testing that the conversion is idempotent. We should also test that the generated wrapper actually works (eg., use the wrapper produced from Sum and make sure it works by exercising all the methods).